frontend/src/main/sign-in/SingInPage.js

```diff
@@ -0,0 +1,49 @@
+import { makeStyles, Paper, Typography } from '@material-ui/core'
+import MainPaper from 'main/MainPaper'
+import React from 'react'
+import { Link } from 'react-router-dom'
+import SignInFormContainer from './SignInFormContainer'
+
+const useStyles = makeStyles((theme) => ({
+    paper: {
```

Comment:
please consider using the following order for styles props if they are present:

1. position
2. position related (top, left ...) 
3. display
4. display related (flex dir, justify, align ...)
5. size related (width, height, margins, borders)
6. paddings
7. backgrounds, shadows, other graphic related
7. inner text styles (color, align, font-weight ...)
8. animations, transfers (usually animation is provided with different class from other styles)